tests/e2e/network/service_annotations.go

```diff
@@ -195,6 +196,37 @@ var _ = Describe("Service with annotation", func() {
 		Expect(err).NotTo(HaveOccurred())
 		Expect(ret).To(Equal(true), "external ip %s is not in the target subnet %s", ip, newSubnetCIDR)
 	})
+	It("should bound to the specified load balancer from azure-load-balancer-mode [MultipleAgentPools][VMSS]", func() {
+		mode_name := ""
+		annotation := map[string]string{
+			azure.ServiceAnnotationLoadBalancerMode: mode_name,
+		}
+
+		service := createLoadBalancerServiceManifest(cs, serviceName, annotation, labels, ns.Name, ports)
```

Comment:
Please randomize the serviceName, e.g. `lbModeLoadBalancerName := serviceName  + string(uuid.NewUUID())`

Reply:
And please move L205-208 into beginning of validateLoadBalancerBackendPools(), as well as the above random service name.